packages/console/src/components/AppContent/index.module.scss

```diff
@@ -18,7 +18,7 @@
 .main {
   flex-grow: 1;
   padding-right: _.unit(6);
-  overflow-y: auto;
+  overflow-y: scroll;
```

Comment:
I don't think this is the most ideal way to do it. Setting it to `scroll` will always show the scrollbar track.
Currently it seems to be OK because the track of the scrollbar is transparent, but the track still takes space and it will affect. the layout of all scrollable contents.

Reply:
This, however, can be a temporary solution, though. If we really want to get rid of the jumpy jittering behavior

Reply:
per discussion: be a temporary solution
https://silverhand-io.slack.com/archives/C0243P3E4KS/p1656556696334429